test/powershell/Modules/Microsoft.PowerShell.Management/Start-Process.Tests.ps1

```diff
@@ -105,17 +105,13 @@ Describe "Start-Process" -Tags @("Feature") {
         $process | Stop-Process
     }
 
-    It "Should be able to use the whatif switch without error" {
-        { Start-Process $pingCommand -ArgumentList $pingParam -WhatIf } | Should Not Throw
-    }
-
-    It "Should be able to use the whatif switch without performing the actual action" {
+    It "Should be able to use the -WhatIf switch without performing the actual action" {
         $pingOutput = Join-Path $TestDrive "pingOutput.txt"
-        Start-Process -Wait $pingCommand -ArgumentList $pingParam -RedirectStandardOutput $pingOutput -WhatIf
+        { Start-Process -Wait $pingCommand -ArgumentList $pingParam -RedirectStandardOutput $pingOutput -WhatIf} | Should Not Throw
```

Comment:
I believe `-ErrorAction Stop` is required or it will never actually throw.

Reply:
I agree.

Reply:
Thanks @markekraus for pointing this out. Not sure how I missed it.